src/main/java/org/folio/circulation/domain/InstanceRequestRelatedRecords.java

```diff
@@ -14,6 +15,7 @@
   private List<Item> itemsWithoutLoans;
   private List<Item> itemsWithoutRequests;
   private RequestByInstanceIdRequest instanceLevelRequest;
+  private Map<Item, RequestQueue> itemRequestQueueMap;
```

Comment:
Should this be part of `Item` someday?

Reply:
I thought @marc-johnson says it's not that useful to have request queues as part of item, referring to a comment on a PR a few weeks ago when I attempted but failed to attach request queues to items due to circular references.

Reply:
OK. Didn't recall that. It just seems naturally part of an `Item` to me, at least in the circulation sense. :)

Reply:
I think mod-circulation has a challenge in its domain modelling where too many parts of a entity can be null or only relevant in certain contexts. 

I think the entities may need shrinking rather than growing.

Making every item have to include a request queue suggests that the request queue is relevant everywhere an item is relevant. Maybe that’s the case, and every time we use an item we should get the request queue for it.

Reply:
If it's relevant to have request queues in an item then perhaps we can make them lazy loaded to not performance worse.

Reply:
We could potentially add lady loading. That would mean an item either needs wrapping in a transparent proxy (like hibernate does) or an item would need to know how to fetch a queue

Reply:
At some point I’m going to experiment with smaller aggregates in the domain